src/cdk/explorer/nodes/constructNode.ts

```diff
@@ -47,13 +50,22 @@ export class ConstructNode extends AWSTreeNodeBase {
         }
     }
 
-    public async getChildren(): Promise<(ConstructNode)[]> {
-        const entities = []
+    public async getChildren(): Promise<(ConstructNode | PropertyNode)[]> {
+        const entities: (ConstructNode | PropertyNode)[] = []
+
+        // add all properties
+        if (this.properties) {
+            const propertyNodes: PropertyNode[] = generatePropertyNodes(this.properties)
+            for (const property of propertyNodes) {
+                entities.push(property)
+            }
```

Comment:
This can become `entities.push(...propertyNodes)`

Reply:
done